Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -493,6 +493,9 @@
 ### LS2 - MWGR 2021 - CSC, DAQ, DCS, DQM, DT, ECAL, GEM, HCAL, L1SCOUT, PIXEL, RPC, TCDS, TRACKER, TRG ###
 steps['RunCosmics2021']={'INPUT':InputInfo(dataSet='/ExpressCosmics/Commissioning2021-Express-v1/FEVT',label='2021GR5',ls={343498: [[1, 10000]]},events=100000,location='STD')}
 
+#### run3 Cosmics ####
+##Run 344518 @ 0T 2021CRUZET
+steps['RunCosmics2021CRUZET']={'INPUT':InputInfo(dataSet='/Cosmics/Commissioning2021-v1/RAW',label='2021CRUZET',events=100000,location='STD')}
```

Comment:
@francescobrivio doesn't this miss the `,ls={344518: [[1, 10000]]},` part?

Reply:
I copied it from the Run2 cosmics case: https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_steps.py#L487-L491 which don't have limitations on the number of ls, only the MWGR has it...not sure why!

Reply:
but wait the Run-2 line has `,run=[256259]`, I dont see the equivalent of that in here

Reply:
ah good catch! I must have deleted it by mistake when I removed the ls range I was using for test. Fix coming right away...